pkg/webhook/ingress/types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package ingress
 
 import (
+	"github.com/go-logr/logr"
```

Comment:
please, sort imports:

1. standard library
2. external packages
3. internal ones (`github.com/clastix/capsule`)

Reply:
It should be after the imports of `k8s.io` then? There are no standard go libraries in this file.

Reply:
My bad, this snippet is good: I was rather pointing to [this](https://github.com/clastix/capsule/pull/162/files/47a20ab0f0705f14d8a0144355a5b40148ba70b3#r542397019)